http-svr/xmlrpc_client.ml

```diff
@@ -357,6 +357,15 @@ module XMLRPC = struct
   let request_to_short_string x = x.Rpc.name
 end
 
+module JSONRPC = struct
+  type response = Rpc.response
+  let response_of_string x = Jsonrpc.response_of_string x
```

Comment:
This works, but could've also just been `let response_of_string = Jsonrpc.response_of_string`